src/test/java/com/plaid/client/integration/PaymentGetTest.java

```diff
@@ -0,0 +1,32 @@
+package com.plaid.client.integration;
+
+import com.plaid.client.PlaidClient;
+import com.plaid.client.request.PaymentGetRequest;
+import com.plaid.client.response.PaymentCreateResponse;
+import com.plaid.client.response.PaymentGetResponse;
+import com.plaid.client.integration.PaymentCreateTest;
+import org.junit.Test;
+import retrofit2.Response;
+
+import static org.junit.Assert.assertNotNull;
+
+public class PaymentGetTest extends AbstractIntegrationTest {
+
+  @Test
+  public void testPaymentGetSuccess() throws Exception {
+
+    Response<PaymentCreateResponse> createPaymentResponse = PaymentCreateTest.createPayment(client());
+    String paymentId = createPaymentResponse.body().getPaymentId();
+    assertNotNull(paymentId);
+
+    Response<PaymentGetResponse> getPaymentResponse =
+      client().service().paymentGet(new PaymentGetRequest(paymentId)).execute();
+    assertSuccessResponse(getPaymentResponse);
+    assertNotNull(getPaymentResponse.body().getPaymentId());
```

Comment:
I think we should also be checking for the payment token here

Reply:
⚡️opening a new PR to address this